lib/quicly.c

```diff
@@ -5404,6 +5404,8 @@ static int handle_payload(quicly_conn_t *conn, size_t epoch, const uint8_t *_src
             /* slow path */
             --state.src;
             if ((state.frame_type = quicly_decodev(&state.src, state.end)) == UINT64_MAX) {
+                state.frame_type =
+                    QUICLY_FRAME_TYPE_PADDING; /* we cannot signal the offending frame type when failing to decode the frame type */
```

Comment:
Hm. Are we supposed to not report the offending frame type in connection close?

Reply:
This is when we fail to decode the offending frame type. Until now, we have been trying to send UINT64_MAX as an indication of decode failure, but we canot really send that value, so we have actually being sending UINT62_MAX.

Reply:
Where does this returned frame_type get used in case of error? I can't seem to find where it gets used.

Reply:
The value is copied to `*offending_frame_type` at the end of this function, then the callers pass it to `initiate_close`.